To: vim_dev@googlegroups.com
Subject: Patch 7.3.469
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.469
Problem:    Compiler warning for unused argument without some features.
Solution:   Add UNUSED.
Files:	    src/buffer.c


*** ../vim-7.3.468/src/buffer.c	2012-02-22 19:13:00.000000000 +0100
--- src/buffer.c	2012-03-07 22:30:56.000000000 +0100
***************
*** 329,335 ****
      win_T	*win;		/* if not NULL, set b_last_cursor */
      buf_T	*buf;
      int		action;
!     int		abort_if_last;
  {
  #ifdef FEAT_AUTOCMD
      int		is_curbuf;
--- 329,335 ----
      win_T	*win;		/* if not NULL, set b_last_cursor */
      buf_T	*buf;
      int		action;
!     int		abort_if_last UNUSED;
  {
  #ifdef FEAT_AUTOCMD
      int		is_curbuf;
*** ../vim-7.3.468/src/version.c	2012-03-07 20:13:44.000000000 +0100
--- src/version.c	2012-03-07 22:51:54.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     469,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
157. You fum through a magazine, you first check to see if it has a web
     address.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
